modules/core/pom.xml

```diff
@@ -141,6 +132,34 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
+        <executions>
+          <execution>
+            <goals>
+              <goal>shade</goal>
+            </goals>
+            <phase>package</phase>
+            <configuration>
+              <relocations>
+                <relocation>
+                  <pattern>org.apache.thrift</pattern>
+                  <shadedPattern>org.shaded.thrfit</shadedPattern>
```

Comment:
Typo here... but also, the new package should be a subpackage of `org.apache.fluo.core` (or whatever the base package is for this fluo module).